agent/engine/docker_client.go

```diff
@@ -73,6 +75,7 @@ const (
 	removeContainerTimeout  = 5 * time.Minute
 	inspectContainerTimeout = 30 * time.Second
 	removeImageTimeout      = 3 * time.Minute
+	ListPluginsTimeout      = 1 * time.Minute
```

Comment:
lint: needs a comment. You can move this to be with rest of the global const's as well

Reply:
Where do we keep global constants?